lib/awt/awt.ts

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+import { AwtUtils } from './awtUtils';
+import { AvnApiConfig } from '../interfaces';
+import log from 'loglevel';
+
+export class Awt {
+  private splitFeeOptions: { hasPayer: boolean; payerAddress: string };
+  private signerAddress: string;
+  private apiSigner: AvnApiConfig['sign'];
+  private awtToken: string;
+
+  constructor(api: AvnApiConfig, signerAddress: string, options) {
+    this.splitFeeOptions = {
+      hasPayer: options.hasPayer,
+      payerAddress: options.payerAddress
+    };
+
+    this.signerAddress = signerAddress;
+    this.apiSigner = api.sign;
+    this.awtToken;
+  }
+
+  async getToken() {
+    if (!AwtUtils.tokenAgeIsValid(this.awtToken)) {
```

Comment:
are we using `!` or` === false` ? I'm happy with either but we should try and be consistent

Reply:
Good point.  `=== false` might be more explicit. I will update